### PR TITLE
Add Trash tag to organs

### LINKED ISSUE #31577: Organs should fit into janitor trash bag
**Screenshots**

image

### PR DESCRIPTION
**Pull request by lzk228**
## About the PR

title
organs can fit in trash bag

## Why / Balance

to resolve #31577 (i decided to redo parenting a little later)

## Requirements

- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- tweak: Organs can fit into trash bags.

### CODE REVIEW

File: Resources/Prototypes/Body/Organs/human.yml [hunk 1]
```diff
@@ -50,9 +51,6 @@
   - type: Examiner
   - type: BlockMovement
   - type: BadFood
-  - type: Tag
-    tags:
-      - Meat
```
Comment (lzk228) [inline Resources/Prototypes/Body/Organs/human.yml:55]: removed since inherits from parent

**Comment by UbaserB** [comment]
can you check if slime organs are parented

**Comment by lzk228** [comment]
probably
organs

**Comment by slarticodefast** [comment]
This will cause a problem for food sequences, since they inherit all tags of the ingredients. For example a burger with a single organ in it will fit in the trash bag, meaning you can fit a lot of giant burgers into the janitor trash bag. There may be a similar problem with the recycler.

**Comment by lzk228** [comment]
okay i think i will firstly do reparenting

**Pull request closed without merging**